Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.m

```diff
@@ -96,6 +99,21 @@ + (instancetype)vok_addWithDictionary:(NSDictionary *)inputDict forManagedObject
     }
 }
 
++ (void)vok_addWithArrayInBackground:(NSArray *)inputArray completion:(VOKManagedObjectReturnBlock)completion
+{
+    [VOKCoreDataManager importArrayInBackground:inputArray
+                                       forClass:[self class]
+                                     completion:^(NSArray *arrayOfManagedObjectIDs) {
+                                         NSMutableArray *returnArray = [NSMutableArray array];
```

Comment:
I know this is @vokal-isaac's thing: this should (or at least could) use `arrayWithCapacity`